build.gradle

```diff
@@ -28,18 +28,18 @@ plugins {
     // NOTE: KEEP THESE VERSIONS ALIGNED WITH THE DEPENDENCIES BLOCK.
 
     // Apply the Spotless plugin for formatting.
-    id "com.diffplug.spotless" version "6.13.0"
+    id "com.diffplug.spotless" version "6.18.0"
 
     // Apply the Error Prone plugin for static code analysis.
     // NOTE: ENSURE `BaselinePlugin.ERRORPRONE_CORE_VERSION` IS UPDATED AS APPROPRIATE.
-    id "net.ltgt.errorprone" version "3.0.1"
+    id "net.ltgt.errorprone" version "3.1.0"
```

Comment:
@adam-kimpton as per the NOTE above, you need to update `BaselinePlugin` file as well.

Reply:
Done.